addon/styles/_pix-filter-banner.scss

```diff
@@ -32,4 +38,13 @@
       margin-right: 0;
     }
   }
+
+  &__details {
+    color: $grey-60;
+    font-family: $font-roboto;
+    font-weight: $font-medium;
```

Comment:
on dirait que la font-weight est pas prise en compte :
![image](https://user-images.githubusercontent.com/23451175/106116326-1015aa80-6152-11eb-9449-698405f813fe.png)

alors que ça devrait ressembler à ça :
![image](https://user-images.githubusercontent.com/23451175/106116278-055b1580-6152-11eb-94f0-014c33bd2e78.png)

Reply:
Nous ne pouvons pas facilement voir la différence entre le font-weight 400 et 500. Pour avoir une différence claire, je suggère que nous changions le font-weight de détails en semi-bold (600).
What do you think?

Reply:
le 600 est trop violent, le 500 ça me semble bien, mais on dirait qu'il est pas pris en compte. Peut-être demander de l'aide au reste de l'équipe @bpetetot par exemple ?

Reply:
On a un problème dans storybook, c'est qu'il ne charge pas les fonts roboto et open-sans (un bug). donc on ne visualise pas le rendu avec la bonne police (roboto) et par conséquent le bon `font-weight`

Reply:
Ah, ok, bon à savoir. je teste dans Pix Orga alors pour voir si ça fait le bon effet.